homework2/task01/text_counter.py

```diff
@@ -0,0 +1,108 @@
+"""
+Given a file containing text. Complete using only default collections:
+    1) Find 10 longest words consisting from largest amount of unique symbols
+    2) Find rarest symbol for document
+    3) Count every punctuation char
+    4) Count every non ascii char
+    5) Find most common non ascii char for document
+"""
+
+import os
+from collections import Counter
+from typing import List
+
+
+def get_longest_diverse_words(file_path: str) -> List[str]:
+    amount_symbols = []
+    with open(os.path.join(file_path)) as text:

```

Comment:
Sure, I recognised that. After our conversation I didn't change encoding file, because inputted text can be in another codec. Default codec is 'UTF-8', you can check it in code.